### DISCUSSION TITLE
Docker image for ARM

### DISCUSSION THREAD

**Discussion by jplegat** [thumbsup/thumbsup#236] [Q&A] [answered]
Hi,

I've used to run **thumbsup** using Docker on a Raspberry Pi 4 - I did this a long time ago and now I don't really remember how I did it back then... memory's not so good... Did it use to have support for ARM?

Anyway, today I've tried to update the image to the latest and noticed there's no tag for ARM when pulling it from the Hub. Is it possible to build a new Docker image for ARM? Which Dockerfile should I use?

Thanks a lot for your help!

JP

**Comment by rprieto** [comment, marked as answer]
Hi @jplegat, I don't think there ever was an official Docker image for ARM before. Maybe it was a custom one?

Anyway, since last week there is! The new images are published to Github to avoid the new Dockerhub rate-limiting (and simplify releases). You can try the following images which are built for AMD64 and ARM64:

```
docker pull ghcr.io/thumbsup/thumbsup:latest
docker pull ghcr.io/thumbsup/thumbsup:2.15.0
```

**Reply by jplegat** [reply]
Awesome news!!!

Yes, you're right, it was a custom image - jaedle/thumbsup-arm - but it's great to have official support!

Thanks a lot!

Edit:

I just tried to pull the images and it returned the following error:

```
latest: Pulling from thumbsup/thumbsup
no matching manifest for linux/arm/v7 in the manifest list entries
```

```
2.15.0: Pulling from thumbsup/thumbsup
no matching manifest for linux/arm/v7 in the manifest list entries
```

Is it available only for 64-bit?

**Reply by rprieto** [reply]
Hi, yes it was but I just published Arm7 images. Can you try again?

**Reply by jplegat** [reply]
That was super quick!

I've just pulled the new image successfully and ran it to create a new gallery, it worked flawlessly!

Thank you so much! I really appreciate it!
